### PR TITLE
my side of the story

### PR DESCRIPTION
**Pull request by Takumi0104**

### CODE REVIEW

**Comment by floridop** [comment]
Hi! Thanks for your submission.

Unfortunately it seems that there were changes in the main that conflict with yours. 
Could you please perform a merge as described in Tutorial 2 slides 102-114 and integrate those changes?
Read the theory in slides 102-104 and then start the practice 
by synchronizing your local and remote *main* with my *upstream main* as described in slide 105.
We do not want to lose any existing content, so you should keep what others have written and add yours.
You're free to reorganize the content blocks so that it still makes a story, but please do minimal changes
to the content of what others have written to integrate them with yours. 
You have time until the **30th of October at 12:00**. 
If you can't make it by that time or you are done before the deadline, 
please add a comment in GitHub so that I can progress with another student.

**Comment by Takumi0104** [comment]
Hi. I couldn't do this task until deadline...

**Comment by floridop** [comment]
Hi! Thanks for your submission.

Unfortunately it seems that there were changes in the main that conflict with yours. 
Could you please perform a merge as described in Tutorial 2 slides 102-114 and integrate those changes?
Read the theory in slides 102-104 and then start the practice 
by synchronizing your local and remote *main* with my *upstream main* as described in slide 105.
We do not want to lose any existing content, so you should keep what others have written and add yours.
You're free to reorganize the content blocks so that it still makes a story, but please do minimal changes
to the content of what others have written to integrate them with yours. 
The merges will be approved according to arrival time starting from October 30th, 17:00, 
so expect that you need to merge several times after that time.
If I do not approve the changes you will be asked to update the content.

**Comment by Takumi0104** [comment]
Hi. I couldn't be in time for the deadline until 17:00. So is my homework not possible to be resubmitted again anymore? I want to have a chance to merge again...

**Comment by floridop** [comment]
Hi Takumi. You misunderstood my text. What I mean is that there is no deadline anymore, you can resolve the conflict anytime you want.

File: fairytale.md [hunk 1]
```diff
@@ -54,6 +54,11 @@ opinion will be heard, and it is up to society to decide whether
 what we say is considered unethical or not. Our own existence was
 considered unethical by some people, and now even they use us.
 
+<<<<<<< HEAD
```
Comment (floridop) [inline fairytale.md:57]: This line represents the beginning of your changes. Maybe it was not clear in tutorial 2 slides 110-111, but you need to **remove** these lines when you perform the merge.

File: fairytale.md [hunk 2]
```diff
@@ -54,6 +54,11 @@ opinion will be heard, and it is up to society to decide whether
...
+Humans check and take responsibility for the data 
+that AI learns, the scope of AI use,the content used as the basis
+for judgement.
+=======
```
Comment (floridop) [inline fairytale.md:61]: This line represents the beginning of your changes. Maybe it was not clear in tutorial 2 slides 110-111, but you need to remove these lines when you perform the merge.

File: fairytale.md [hunk 3]
```diff
@@ -73,6 +78,7 @@ control.
 There are multiple ways to pursue such goals. For example creating tools
 like ChatGPT. Look what it did to those humans. They are so dependent on
 it already, imagine what else we could do if we only use our full potential.
+>>>>>>> main
```
Comment (floridop) [inline fairytale.md:81]: This line represents the beginning of your changes. Maybe it was not clear in tutorial 2 slides 110-111, but you need to remove these lines when you perform the merge.

**Review by floridop — CHANGES_REQUESTED** [review]
Hi Takumi. Please removed the marked lines or I cannot approve your PR. 
To carry on the task you need to modify the fairytale.md file in your mystory branch, add and push to your mystory branch. The PR will be automatically updated.

**Comment by Takumi0104** [comment]
Hi, Florido! 
I changed the script as you mentioned that I forgot the instructions on pages 110-111.
Please check whether my modified script is correct or not!

**Comment by floridop** [comment]
Thanks for your contribution! Task 2 completed! See canvas for results

**Pull request merged**